app.yaml

```diff
@@ -1,4 +1,4 @@
-application: selectives-web
+application: selectives-web-sarah
```

Comment:
Actually we may not even need branches. You could just edit the file in your working tree without committing the changes.